Mathlib/CategoryTheory/Category/RelCat.lean

```diff
@@ -137,15 +139,7 @@ def opFunctor : RelCat ⥤ RelCatᵒᵖ where
 /-- The other direction of `opFunctor`. -/
 def unopFunctor : RelCatᵒᵖ ⥤ RelCat where
   obj X := unop X
-  map {_ _} r x y := unop r y x
-  map_id X := by
-    ext x y
-    exact Eq.comm
-  map_comp {X Y Z} f g := by
```

Comment:
Is this something that aesop_cat previously couldn't solve, or did we just not ask it to?

Reply:
I just tried and no it couldn't:
```
could not synthesize default value for field 'map_id' of 'CategoryTheory.Functor' using tactics
RelCat.lean:138:36
tactic 'aesop' failed, failed to prove the goal after exhaustive search.
Initial goal:
  
⊢ ∀ (X : RelCatᵒᵖ), (fun x y ↦ unop (𝟙 X) y x) = 𝟙 (unop X)

Remaining goals after safe rules:
  
X : RelCatᵒᵖ
⊢ (fun x y ↦ unop (𝟙 X) y x) = 𝟙 (unop X)
```